protocol/x/clob/abci.go

```diff
@@ -180,20 +186,6 @@ func PrepareCheckState(
 		panic(err)
 	}
 
-	telemetry.ModuleSetGauge(
-		types.ModuleName,
-		float32(len(subaccountIds)),
-		metrics.Liquidations,
-		metrics.LiquidatableSubaccountIds,
-		metrics.Count,
-	)
-
-	telemetry.ModuleSetGauge(
-		types.ModuleName,
-		metrics.GetMetricValueFromBigInt(keeper.GetInsuranceFundBalance(ctx)),
-		metrics.InsuranceFundBalance,
-	)
```

Comment:
I think this is more appropriate in `EndBlocker`

Reply:
Is it acceptable that this will only stat the insurance fund balance in the `deliverState`, and never stat the insurance fund balance of the `checkState`?

Would we ever want to know what each validator thinks the insurance fund balance would be after processing their liquidations?

Reply:
We have insurance fund delta info so we can technically sort of calculate from that.